CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to `WP Curate` will be documented in this file.
 
+## 1.8.3 - 2024-04-04
+
+- Enhancement: Exclude current post in backfilled posts query.
```

Comment:
Since there are no bugfixes in the release, it seems more like a minor release than a patch release to me, i.e. `1.9.0`.